src/player.ts

```diff
@@ -2006,6 +2007,21 @@ export default class Player extends FakeEventTarget {
     }
   }
 
+  /**
+   * In iframe embed add title element with the entry name
+   * @returns {void}
+   * @private
+   */
+  private _addTitleOnIframe(): void {
+    const head = Utils.Dom.getElementBySelector('head');
```

Comment:
how will this match only iframe embeds ?
what if there is no name in metadata ?

Reply:
add the iframe check.
if there is no name title will be empty (per Sagi)